src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -300,6 +303,63 @@ private void testAddQuestionAction() throws Exception {
         feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackQuestionAddSuccess.html");
     }
 
+    private void testAddTemplateQuestionLink() throws Exception {
+
+        ______TS("add template question link");
+
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.clickTemplateModalButton();
+
+        assertFalse("Unable to submit when there is no question selected",
+                feedbackEditPage.isAddTemplateQuestionButtonEnabled());
+    }
+
+    private void testAddTemplateQuestionAction() throws Exception {
+
+        ______TS("add template question action success");
+
+        // check if each question panel is expandable
+        feedbackEditPage.clickTemplateQuestionPanel(1);
+        feedbackEditPage.clickTemplateQuestionPanel(2);
+        feedbackEditPage.clickTemplateQuestionPanel(3);
+        feedbackEditPage.clickTemplateQuestionPanel(4);
+        feedbackEditPage.clickTemplateQuestionPanel(5);
+        feedbackEditPage.verifyHtmlPart(By.id("addTemplateQuestionModal"),
```

Comment:
Instead of doing this why not just assert if a certain class is present when the panel is expanded. Using verifyByHtml doesnt make sense here

Reply:
Oh yes, that is a very good point. I realise that the html part will be the same whether the panel is clicked or not. I will add a check for the `collapse in` class when the panel is expanded. In that case, is it necessary to check all the panels?

Also, I feel that it is important to verify that the content of the template modal is accurate as the content is different from the original `json` format. What do you think?

Reply:
> Also, I feel that it is important to verify that the content of the template modal is accurate as the content is different from the original json format. What do you think?

I agree. Lets place the verifying part for that elsewhere cause its better to have assert checks for the panels collapse than use verifyHTML. Lets do verifyHTML in some other/new test case?